packages/core/src/Table/stories/TableSection/TableSettings.tsx

```diff
@@ -0,0 +1,522 @@
+          (args) => myModifier(bdId || "", args),
+        ]}
+      >
+        {activeColumn && (
+          <ColumnContainer
+            column={activeColumn}
+            tasks={tasks.filter((task) => task.columnId === activeColumn.id)}
+          />
+        )}
+        {activeTask && <TaskCard task={activeTask} />}
+      </DragOverlay>
+    </DndContext>
+  );
+};
+
+export const TableSettings = () => {
+  const tableColumns = useMemo(() => getColumns(), []);
+  const [popperOpen, setPopperOpen] = useState(false);
+  const [data] = useState(makeData(5));
+  const [columnOrder, setColumnOrder] = useState(tableColumns.map((c) => c.id));
+  const [hiddenColumns, setHiddenColumns] = useState<string[]>([]);
+  const [fixedColumns, setFixedColumns] = useState<string[]>([]);
+  const bdId = useUniqueId("base-dropdown");
+
+  const [updatedColumns, setUpdatedColumns] =
+    useState<HvTableColumnConfig<AssetEvent, string>[]>(tableColumns);
+
+  useEffect(() => {
+    const cols = tableColumns.map((column) => {
+      if (fixedColumns.includes(column.accessor as string)) {
+        return { ...column, sticky: "left" };
+      }
+      return column;
+    });
+    setUpdatedColumns(cols);
+  }, [fixedColumns, tableColumns]);
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    prepareRow,
+    headerGroups,
+    page,
+    state: { pageSize },
+    setColumnOrder: setTableColumnOrder,
+    setHiddenColumns: setTableHiddenColumns,
+  } = useHvData<AssetEvent, string>(
+    {
+      columns: updatedColumns,
+      data,
+      initialState: {
+        pageSize: 5,
+        columnOrder: [...fixedColumns, ...columnOrder] as string[],
+        hiddenColumns: hiddenColumns as string[],
+      },
+    },
+    useHvSortBy,
+    useHvPagination,
+    useHvTableSticky,
+    useColumnOrder
+  );
+
+  const renderTableRow = (i: number) => {
+    const row = page[i];
+
+    if (!row) return <EmptyRow key={i} />;
+
+    prepareRow(row);
+
+    return (
+      <HvTableRow {...row.getRowProps()}>
+        {row.cells.map((cell) => (
+          <HvTableCell {...cell.getCellProps()}>
+            {cell.render("Cell")}
+          </HvTableCell>
+        ))}
+      </HvTableRow>
+    );
+  };
+
+  const handleOrderChange = (order) => {
+    setColumnOrder(order);
+    setTableColumnOrder?.(order);
+  };
+
+  const handleHideColumn = (columnId: string, operation: "show" | "hide") => {
+    if (operation === "hide") {
+      const newHiddenColumns = [...hiddenColumns, columnId];
+      setHiddenColumns(newHiddenColumns);
+      setTableHiddenColumns?.(newHiddenColumns);
+    } else if (operation === "show") {
+      const newHiddenColumns = hiddenColumns.filter((c) => c !== columnId);
+      setHiddenColumns(newHiddenColumns);
+      setTableHiddenColumns?.(newHiddenColumns);
+    }
+  };
+
+  return (
+    <HvTableSection
+      title={<HvTypography variant="title4">Events</HvTypography>}
+      actions={
+        <HvBaseDropdown
```

Comment:
When we hover the button, the button jumps because a border is added. I think we should remove the border.